src/views/main.js

```diff
@@ -36,6 +36,12 @@ const mainLayout = data => `
     <link rel='apple-touch-icon' href='/images/apple-touch-icon.webp' sizes='180x180'>
 
     <script src='/js/index.js' type='module'></script>
+    ${
+      data.partial.name === 'emailPreview'
+        ? '<script src="/js/partials/email-preview.js" type="module"></script>'
+        : ''
+    }
+
```

Comment:
Instead of adding a conditional block into the main template for this script, why not statically add it directly in the partial?  See suggestion below.

Reply:
Yes — let’s do it like that.